R/assert.R

```diff
@@ -2,13 +2,41 @@
 #'
 #' @description
 #'
-#' * `vec_is()` is a predicate that checks if its input conforms to a
-#'   prototype and/or a size.
-#'
-#' * `vec_assert()` throws an error when the input doesn't conform.
+#' * `vec_is()` is a predicate that checks if its input is a vector that
+#'   conforms to a prototype and/or a size.
+#'
+#' * `vec_assert()` throws an error when the input is not a vector or
+#'   doesn't conform.
+#'
+#' @section Scalars and vectors:
+# From vec_base_typeof():
```

Comment:
`vec_base_typeof()` is not quoted. But I don't think it should be mentioned here.